### ISSUE TITLE
RW TURP: Property & Street Information tab gives JAVA Error when adding and then saving or just saving multiple new properties

### ISSUE THREAD

**Issue by TracyLinder** [cityofaustin/atd-AMANDA-V7#74]
RSN: 12591822

![image.png](https://images.zenhubusercontent.com/5cca1e522b8dfc0ee14dedf5/125b0759-041e-4702-9424-e0e713831786)

**Comment by Nadin-Nader** [comment]
**What was happening:**
The user would go to a/multiple properties on the Property and Segment Tab to add the meter/umetered information on a Lane Closures or whichever info type was needed, they would save and get a java exception error. This was done with the meter/umetered information on a Lane Closures or whichever info type was neede, with a date and with comments (this was also done with each individually) and they still go the Java error.

**What is Happening Now:**
Went back in on Test and tried to add a few properties in the Property and Segment tab. I added 4500-4553 Mattie ST segment and 4524 Mattie St with the dimensions needed and i did not get an error.

I used the Traffic Lane : Dimensions and Parking Lane : Dimensions info types. Added the width for both to 40 the length and 10 in length for Traffic Lane : Dimensions. I set the date for Jan 22 2021 and saved successfully. 

I then tried to add more, i was able to add 4 additional ones (i deleted them) without the Exception error multiple times. I added them with comments. I tried with "Number of #" field and all was successful. 
Tracy will verify.

This may have been resolved
